index.Rmd

```diff
@@ -38,6 +39,7 @@ library(caret)
 library(forcats)
 library(sp)
 library(raster)
+library(csvread)
```

Comment:
Call all packages at beginning of your code so that others may see what packages they need without searching through the rest of your code.